src/components/sendV2/summary/summary.js

```diff
@@ -71,38 +127,41 @@ class Summary extends React.Component {
     }
   }
 
-  amountAndFee() {
-    return fromRawLsk(toRawLsk(this.props.fields.amount.value) + fees.send);
-  }
-
   prevStep() {
     Piwik.trackingEvent('Send_Summary', 'button', 'Previous step');
     this.props.prevStep({ fields: { ...this.props.fields } });
   }
 
   nextStep() {
     Piwik.trackingEvent('Send_Summary', 'button', 'Next step');
-    this.props.sent({
-      account: this.props.account,
-      recipientId: this.props.fields.recipient.address,
-      amount: this.amountAndFee(),
-      data: this.props.fields.reference.value,
-      passphrase: this.props.account.passphrase,
-      secondPassphrase: this.state.secondPassphrase.value,
+    this.submitTransaction();
+    this.props.nextStep({
+      fields: {
+        ...this.props.fields,
+        hwTransactionStatus: false,
+        isHardwareWalletConnected: false,
+      },
     });
-    this.props.nextStep({ fields: { ...this.props.fields } });
   }
 
   render() {
-    const { secondPassphrase } = this.state;
-    const isBtnDisabled = secondPassphrase.hasSecondPassphrase
+    const { secondPassphrase, isHardwareWalletConnected } = this.state;
+    let isBtnDisabled = secondPassphrase.hasSecondPassphrase
       ? secondPassphrase.isValid !== '' && !secondPassphrase.isValid
       : false;
+    isBtnDisabled = !isBtnDisabled && isHardwareWalletConnected;
+
+    const confirmBtnMessage = isHardwareWalletConnected
+      ? 'Confirm on Ledger'
+      : `Send ${this.props.fields.amount.value} LSK`;
+
+    const title = isHardwareWalletConnected ? 'Confirm transaction on Ledger Nano S' : 'Transaction summary';
+
 
     return (
       <div className={`${styles.wrapper} summary`}>
         <header className={`${styles.header} summary-header`}>
-          <h1>{this.props.t('Transaction summary')}</h1>
+          <h1>{this.props.t('{{title}}', { title: `${title}` })}</h1>
```

Comment:
With the above suggestion you would not need the `this.props.t` here, just use the `title` variable

Reply:
done